pysat/tests/test_utils.py

```diff
@@ -84,375 +109,351 @@ def test_initial_pysat_load(self, capsys):
+        return
 
 
 class TestScaleUnits():
+    """Unit tests for `scale_units`."""
     def setup(self):
         """Runs before every method to create a clean testing setup."""
         self.deg_units = ["deg", "degree", "degrees", "rad", "radian",
                           "radians", "h", "hr", "hrs", "hours"]
         self.dist_units = ["m", "km", "cm"]
         self.vel_units = ["m/s", "cm/s", "km/s", 'm s$^{-1}$', 'cm s$^{-1}$',
                           'km s$^{-1}$', 'm s-1', 'cm s-1', 'km s-1']
+        self.scale = 0.0
 
     def teardown(self):
         """Runs after every method to clean up previous testing."""
-        del self.deg_units, self.dist_units, self.vel_units
+        del self.deg_units, self.dist_units, self.vel_units, self.scale
+
+    def eval_unit_scale(self, out_unit, scale_type):
+        """Evaluate the unit scaling.
+
+        Parameters
+        ----------
+        out_unit : str
+            Output unit name string
+        scale_type : str
+            String specifying 'angles' or 'distance'
+
+        """
+
+        if scale_type.lower() == 'angles':
+            if out_unit.find("deg") == 0:
+                assert self.scale == 1.0
+            elif out_unit.find("rad") == 0:
+                assert self.scale == np.pi / 180.0
+            else:
+                assert self.scale == 1.0 / 15.0
+        elif scale_type.lower() == 'distance':
+            if out_unit == "m":
+                assert self.scale == 1.0
+            elif out_unit.find("km") == 0:
+                assert self.scale == 0.001
+            else:
+                assert self.scale == 100.0
+        elif scale_type.lower() == 'velocity':
+            if out_unit.find("m") == 0:
+                assert self.scale == 1.0
+            elif out_unit.find("km") == 0:
+                assert self.scale == 0.001
+        return
 
     def test_scale_units_same(self):
-        """ Test scale_units when both units are the same """
+        """ Test scale_units when both units are the same."""
 
-        scale = pysat.utils.scale_units("happy", "happy")
+        self.scale = pysat.utils.scale_units("happy", "happy")
 
-        assert scale == 1.0
+        assert self.scale == 1.0
+        return
 
     def test_scale_units_angles(self):
-        """Test scale_units for angles """
+        """Test scale_units for angles."""
 
         for out_unit in self.deg_units:
-            scale = pysat.utils.scale_units(out_unit, "deg")
-
-            if out_unit.find("deg") == 0:
-                assert scale == 1.0
-            elif out_unit.find("rad") == 0:
-                assert scale == np.pi / 180.0
-            else:
-                assert scale == 1.0 / 15.0
+            self.scale = pysat.utils.scale_units(out_unit, "deg")
+            self.eval_unit_scale(out_unit, 'angles')
+        return
 
     def test_scale_units_dist(self):
-        """Test scale_units for distances """
+        """Test scale_units for distances."""
 
```

Comment:
```suggestion
```

Reply:
This makes the tests harder to read, so I am not committing it.